plugin/src/main/kotlin/com/github/triplet/gradle/play/PlayPublisherExtension.kt

```diff
@@ -40,6 +41,27 @@ open class PlayPublisherExtension : AccountConfig by PlayAccountConfigExtension(
      */
     var errorOnSizeLimit = true
 
+    internal var _resolutionStrategy = ResolutionStrategy.IGNORE
```

Comment:
We should default to FAIL here as that would keep the original behavior of 1.x. And I think it makes sense, ignoring upload errors should be a conscious decision